examples/generative/corrdiff/train.py

```diff
@@ -265,6 +269,8 @@ def main(cfg: DictConfig) -> None:
     )
 
     (img_shape_y, img_shape_x) = dataset.image_shape()
+    img_shape_x = dataset_cfg["img_shape_x"]
+    img_shape_y = dataset_cfg["img_shape_y"]
```

Comment:
What is wrong with "image_shape"? Should it be fixed?

Reply:
Nothing wrong. I forget that img_shape has been defined. Deleted 272 and 273 and resolving this conversation. Thanks